admin/src/utils/parseTeamIdFromFigmaUrl.test.ts

```diff
@@ -0,0 +1,24 @@
+import { parseTeamIdFromFigmaUrl } from './parseTeamIdFromFigmaUrl';
+
+describe('parseTeamIdFromFigmaUrl', () => {
+	// eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
@akostevich-atlassian @Roystbeef I suspect there's something odd with the TS configuration for the admin/ folder. I couldn't figure out what it was, so I'm suppressing the lint checks for the time-being.

Reply:
@elainefigma , I've resolved the issue with a temporary workaround.

@Roystbeef , @elainefigma , in order to prevent these tricky issues, I highly recommend separating backend and frontend projects into separate folders and avoid sharing configuration (e.g., for ESLint, Jest, etc.) between them. The frontend and backend projects are very different and might be very difficult to maintain setup that fits both of them.

```
/figma-for-jira
├─ frontend/
│  ├─ src/
│  ├─ eslintrc.cjs
│  ├─ package.json
│  ├─ tsconfig.json
│  ├─ ...
├─ backend/
│  ├─ src/
│  ├─ eslintrc.cjs
│  ├─ package.json
│  ├─ tsconfig.json
│  ├─ ...
├─ docker-compose.yml
├─ package.json
├─ README.md
├─ ...
``` 

What do you think?

Reply:
yeah that's a good call